modules/transport-netty4/src/main/java/org/elasticsearch/http/netty4/Netty4HttpRequest.java

```diff
@@ -41,59 +41,36 @@ public class Netty4HttpRequest implements HttpRequest {
 
     private final FullHttpRequest request;
     private final BytesReference content;
-    private final HttpHeadersMap headers;
+    private final Map<String, List<String>> headers;
```

Comment:
Changed in order for the `Netty4HttpRequest` to (continue to ) implement the `HttpRequest` which extends the new `BasicHttpRequest` interface.